src/index.css

```diff
@@ -28,54 +27,165 @@ code {
   margin-top: 10px;
   font-weight: 100;
   padding-block: 15px;
-
+  position: relative;
+  margin-inline: 15px;
 }
-.buton{
+.buton {
   margin-left: 15px;
-  height:35px;
-  font-size: small; 
-  
+  height: 30px;
+  font-size: smaller;
+}
+.buton-input {
+  height: 30px;
+  font-size: smaller;
+  background-color: #3b82f6;
+  border: none;
+  margin-top: 2px;
 }
-.p-inputtext{
+
+.buton-check {
+  height: 30px;
+  font-size: smaller;
+  background-color: #3b82f6;
+  border: none;
+  margin-top: 2px;
+  width: 100px !important;
+  padding-inline: 8px;
+  margin-left: 5px;
+}
+.class > .p-inputtext {
   font-size: small;
+  margin-top: -15px;
 }
-.p-dropdown-panel .p-dropdown-items .p-dropdown-item{
+.p-inputtext {
   font-size: small;
+  margin-top: -8px;
 }
-.p-datatable-wrapper{
+.p-dropdown-panel .p-dropdown-items .p-dropdown-item {
   font-size: small;
 }
-.p-fieldset .p-fieldset-legend{
+.p-datatable-wrapper {
+  font-size: small;
+  margin: -5px;
+}
+.p-fieldset .p-fieldset-legend {
   background-color: transparent;
   border: none;
   padding: 5px;
-
 }
-.p-fieldset-legend > span{
+.p-fieldset-legend > span {
   font-weight: lighter;
   font-size: small;
 }
-.input-style{
-width: 290px;
-height: 35px;
-margin-block: 5px;
+.input-style {
+  height: 30px;
+  margin-block: 5px;
+  font-size: small;
 }
-
-.input-style:disabled{
+.input-style:disabled {
   background-color: #f3f4f6;
-  }
+}
+.disabled-background {
+  background-color: #f3f4f6;
+}
 
-.p-fieldset .p-fieldset-content{
+.p-fieldset .p-fieldset-content {
   padding: 0;
 }
 
-.p-tieredmenu .p-menuitem > .p-menuitem-content .p-menuitem-link .p-menuitem-text{
+.p-tieredmenu
+  .p-menuitem
+  > .p-menuitem-content
+  .p-menuitem-link
+  .p-menuitem-text {
   font-size: small;
 }
-.p-card .p-card-content{
+.p-card .p-card-content {
   padding: 10px;
 }
-.p-card .p-card-body{
+.p-card .p-card-body {
   padding: 0px;
 }
+.span-style {
+  width: 40%;
+}
+
+.p-confirm-dialog-accept {
+  margin-left: 15px;
+  height: 35px;
+  font-size: small;
+  margin-top: 20px;
+}
+
+.p-confirm-dialog-reject {
+  margin-left: 15px;
+  height: 35px;
+  font-size: small;
+}
+/* 768 */
```

Comment:
why that ?

Reply:
when I do my tests, I sometimes forget to remove my comments